lib/node/Creator.js

```diff
@@ -48,12 +48,12 @@ const NodeCreator = {
   // exponent might be null, which means there's no exponent node.
   // similarly, coefficient might be null, which means there's no coefficient
   // the symbol node can never be null.
-  polynomialTerm (symbol, exponent, coeff, coeffIsNegOne=false) {
+  polynomialTerm (symbol, exponent, coeff, coeffIsOne=false, coeffIsNegOne=false) {
```

Comment:
I wonder if we could combine `coeffIsOne` and `coeffIsNegOne` to a shared variable... something about explicit coefficients

or is that more confusing?